openapi/geonames/Ballerina.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+distribution = "slbeta6"
+license = ["Apache-2.0"]
+keywords = ["IT Operations/Geographic Information Systems", "Cost/Freemium"]
+org = "ballerinax"
+name = "geonames"
+icon = "icon.png"
+repository = "https://github.com/ballerina-platform/ballerinax-openapi-connectors"
+version = "1.0.1"
```

Comment:
Shall we go with version 1.0.0 since early connector PRs have used the same version as the initial one. In case if this is updated please make sure to rebuild & update the Dependencies.toml too.

Reply:
Updated to 1.0.0 in https://github.com/ballerina-platform/openapi-connectors/commit/c7460483171e99efbe00f5712f1426da695c5445. Thanks